packages/@ngtools/webpack/src/angular_compiler_plugin.ts

```diff
@@ -1056,9 +1056,11 @@ export class AngularCompilerPlugin {
         if (locales.indexOf(parentLocale) !== -1) {
           locale = parentLocale;
         } else {
-          throw new Error(
-            `Unable to load the locale data file "@angular/common/locales/${locale}", ` +
-            `please check that "${locale}" is a valid locale id.`);
+          this._warnings.push(`AngularCompilerPlugin: Unable to load the locale data file ` +
+            `"@angular/common/locales/${locale}", ` +
+            `please check that "${locale}" is a valid locale id.
+            If needed, localeData can be registered against custom locale in application module.`);
```

Comment:
`If needed, localeData can be registered against custom locale in application module.` --> `If needed, you can use "registerLocaleData" manually.`

Reply:
Sure will do  it asap.

Reply:
you still need to change this text :)